tests/unit/legate.core/test_store.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 NVIDIA Corporation
```

Comment:
@bryevdv I go ahead of you and add one unit test for Store. Hopefully this is in line with your plan.